### ISSUE TITLE
ListAssistants returning 400 error because of content-type

### ISSUE THREAD

**Issue by weaseldotro** [sashabaranov/go-openai#660]
I'm getting the following error when calling `ListAssistants`
```
error, status code: 400, message: Unsupported content type: 'application/json; charset=utf-8'. This API method only accepts 'application/json' requests, but you specified the header 'Content-Type: application/json; charset=utf-8'. Please try again with a supported content type.
```

It looks like removing `; charset=utf-8` from the content type header inside the `sendRequest()` function in `client.go` fixes it.

**Comment by str1t3r** [comment]
Got the same issue with `CreateAssistant` method. I've also found [this](https://community.openai.com/t/error-while-trying-to-create-new-thread/624295/2) topic, it seems to be common problem today with community openai libs.

**Comment by raphoester** [comment]
This issue should be resolved because it has been fixed there #659
